.gitignore

```diff
@@ -103,3 +103,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
```

Comment:
oops ? Were we to commit this file ?

Reply:
You mean that black space over there @HawiCaesar ?

Reply:
Let me rephrase. Is the file `staticfiles` going to be in all our directories if this PR is merged and hence you adding it to .gitignore ?

Reply:
Yes @HawiCaesar . When you run `python manage.py runserver` it will autogenerate the static files when the PR is merged. They are over 600 files so that's why we can't have them here

Reply:
*`collectstatic`

Reply:
eh, alright keep that